src/layers/RouteLayer/RouteLayer.js

```diff
@@ -26,7 +28,12 @@ class RouteLayer extends CasaLayer {
       }),
       ...options,
     });
-    this.set('showPopupOnHover', true);
+    this.set('showPopupOnHover', features => {
```

Comment:
```suggestion
    this.set('showPopupOnHover', (features = []) => {
```
to make sure it's always an array